ballerina/Ballerina.toml

```
@@ -2,6 +2,10 @@
 org = "ballerinax"
 name = "oracledb"
 version = "1.0.1"
+authors = ["Ballerina"]
+keywords = ["database", "network", "SQL", "RDBMS", "OracleDB"]
```

```suggestion
keywords = ["database", "network", "SQL", "RDBMS", "OracleDB", "Oracle"]
```
Many people may search with word `Oracle`.